corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -1044,7 +1044,7 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
 
         self.onClear = function () {
             SignatureEntry.prototype.onClear.call(self);
-            self.signaturePad.clear();
+            self.signaturePad && self.signaturePad.clear();
```

Comment:
That's accurate and fair. This is just making sure `self.signaturePad` exists, otherwise it fails when `onClear` is called on loading an incomplete form with a signature question. https://github.com/dimagi/commcare-hq/pull/33481/commits/1cefe1ec6d1f3fc8dd65cd4af9bd35b5c7f90cff